### PR TITLE
[breaking-change] Updated default text styles for the menus

### PR DESCRIPTION
**Pull request by TahaTesser**
Breaking change migration for https://github.com/flutter/flutter/pull/131930

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by TahaTesser** [comment]
cc: @rydmike 
Please let me know if this looks correct and if you've any suggestions.

**Review by parlough — COMMENTED** [review]
Thanks for always being on top of these breaking change/migration guides.

I haven't taken a close look yet, but can you add this to "Not yet released to stable" in the index: https://github.com/flutter/website/blob/main/src/release/breaking-changes/index.md?plain=1#L32

File: src/release/breaking-changes/menus-text-style.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+),
+```
+
+```dart
+menuButtonTheme: MenuButtonThemeData(
+  style: MenuItemButton.styleFrom(
+    textStyle: Theme.of(context).textTheme.bodyLarge,
+  ),
+),
+
```
Comment (parlough) [inline src/release/breaking-changes/menus-text-style.md:167]: This code block needs to be closed.

````suggestion
```

````

File: src/release/breaking-changes/menus-text-style.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+The default text styles used for the menus are updated
+to match the Material 3 specification.
+
+## Context
+
+The default text style for `MenuItemButton`, a widget for use
+in a `MenuBar`, in a menu created with `MenuAnchor`, and `DropdownMenuEntry`
+in the `DropdownMenu` is updated to match the Material 3 specification.
+
+The default text style for the `DropdownMenu`s `TextField` is updated to match
```
Comment (sfshaza2) [inline src/release/breaking-changes/menus-text-style.md:19]:
```suggestion
Likewise, the default text style for the `DropdownMenu`s `TextField` is updated to match
```

**Review by sfshaza2 — APPROVED** [review]
lgtm with a couple minor edits. @TahaTesser, is this part of the stable release for 3.13? You link to the current API, rather than the master channel API. Have these changes already been landed in the API docs?

**Comment by Piinks** [comment]
> is this part of the stable release for 3.13? You link to the current API, rather than the master channel API. Have these changes already been landed in the API docs?

@sfshaza2 No, the change has not landed. It is linked in the description: https://github.com/flutter/flutter/pull/131930

File: src/release/breaking-changes/menus-text-style.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+```dart
+menuButtonTheme: MenuButtonThemeData(
+  style: MenuItemButton.styleFrom(
+    textStyle: Theme.of(context).textTheme.bodyLarge,
+  ),
+),
+
+## Timeline
+
+Landed in version: TBD<br>
```
[the remark this answers is not in the document]
Comment (Piinks) [reply src/release/breaking-changes/menus-text-style.md:170]: It has not landed yet, so there is no build tag.

**Comment by TahaTesser** [comment]
> Thanks for always being on top of these breaking change/migration guides!
> 
> I haven't taken a close look yet, but can you add this to "Not yet released to stable" in the index: https://github.com/flutter/website/blob/main/src/release/breaking-changes/index.md?plain=1#L32

I think we can add to the "Not yet released to stable" section when https://github.com/flutter/flutter/pull/131930 lands and we've a build tag.

File: src/release/breaking-changes/menus-text-style.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+## Summary
+
+The default text styles used for the menus are updated
+to match the Material 3 specification.
+
+## Context
+
+The default text style for `MenuItemButton`, a widget for use
+in a `MenuBar`, in a menu created with `MenuAnchor`, and `DropdownMenuEntry`
+in the `DropdownMenu` is updated to match the Material 3 specification.
```
[the remark this answers is not in the document]
Comment (TahaTesser) [reply src/release/breaking-changes/menus-text-style.md]: it's `MenuBar` and `MenuAnchor` so I added `and` in your suggestion.

**Comment by TahaTesser** [comment]
https://github.com/flutter/flutter/pull/131930 landed last night however we don't have the version tag yet.

**Comment by parlough** [comment]
Thanks for the update! Should be ready to land this soon then :D

**Review by parlough — APPROVED** [review]
Thank you. Looks good to me!

I added the "Landed in" version and added the page to the index.

**Comment by TahaTesser** [comment]
@parlough Appreciate it!

**Pull request merged**